lib/ldp-middleware.js

```diff
@@ -24,7 +24,7 @@ function LdpMiddleware (corsSettings) {
   router.copy('/*', acl.allow('Write'), copy)
   router.get('/*', index, acl.allow('Read'), get)
   router.post('/*', acl.allow('Append'), post)
-  router.patch('/*', acl.allow('Write'), patch)
+  router.patch('/*', acl.allow('Append'), patch)
```

Comment:
@dmitrizagidulin This is a permission downgrade that, with other patch handlers, could allow unintended access. The patch handler is assumed to check for _Write_ (and _Read_) permission when needed after parsing the patch.

(Note that this handler did not check for _Read_ yet.)